.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - test/**
```

Comment:
This should include `release/` branches as well, and they should be included in the conditions controlling whether to deploy.  We don't always push release branches when we're doing a release for this project, but if we did we would want them to build.

Reply:
adding.